homeassistant/components/ezviz/strings.json

```
@@ -68,6 +68,16 @@
           "intensive": "Intensive",
           "silent": "Silent"
         }
+      },
+      "battery_camera_work_mode": {
+        "name": "Work Mode",
```

```suggestion
        "name": "Work mode",
```
Should it mention the battery by the way?